docs: docling integration into file component (#9481)
langflow-ai/langflow: docs/docs/Support/release-notes.mdx

### New features and enhancements

> - Manage dependencies in the UI

Author: This feature will not be included in 1.6, but the Docling dependency is included with the Langflow OSS package.

  The **Create knowledge** component ingests knowledge into a knowledge base. The **Retrieve knowledge** component is added to flows to access a knowledge base.
  For more information, see Knowledge bases.

- Advanced parsing available for the **File** component

>   The **File** component includes advanced parsing capabilities powered by the Docling library.

Reviewer: ```suggestion
  The **File** component supports [advanced parsing with the Docling library](/components-data#advanced-parsing).
```

- Built-in Docling support

>     The Docling dependency is now included with the Langflow OSS package.
>     You don't need to manually install it to use the Docling components or the File component's advanced parsing feature.

Reviewer: ```suggestion
    The Docling dependency is now included with Langflow, making it easier to use the **Docling** components and the **File** component's new advanced parsing feature.
```
